src/styles/card.scss

```diff
@@ -0,0 +1,153 @@
+@import './variables';
+
+.card {
+  box-sizing: border-box;
+  align-items: center;
+  width: 200px;
+  background-color: white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  font-family: Roboto, sans-serif;
+  transition: transform 300ms ease-in-out;
+
+  &:hover {
+    transform: scale(1.2); // Збільшення карти на 20%
+  }
+
+  &__image {
+    box-sizing: border-box;
+    background-image: url(../images/imac.jpeg);
+    width: 160px;
+    height: 134px;
+    background-size: cover;
+    margin: 0 auto;
+  }
+
+  &__product-name {
+    font-family: Roboto, sans-serif;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    text-align: left;
+    color: #{$main-accent};
+    width: 100%;
+    margin-top: 40px;
+    margin-bottom: 0;
+  }
+
+  &__product-id {
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: left;
+    color: #{$secondary};
+    margin-top: 4px;
+  }
+
+  &__title {
+    transition: color 300ms ease-in-out;
+  }
+
+  &:hover .card__title {
+    color: #34568b; // Зміна кольору тексту заголовка на колір #34568b
+  }
+
+  &__reviews {
+    display: flex;
+    align-items: flex-end;
+    justify-content: space-between;
+    margin-top: 16px;
+  }
+
+  &__stars {
+    display: flex;
+    margin: 0;
+  }
+
+  &__star {
+    background-image: url(../images/star.svg);
+    width: 16px;
+    height: 16px;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:nth-child(-n + 4) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &__reviews-label {
+    font-family: Roboto, sans-serif;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    text-align: right;
+    color: #{$main-accent};
+    margin: 0;
+  }
+
+  &__price {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-top: 24px;
+  }
+
+  &__price-label {
+    margin: 0;
+  }
+
+  &__price-label--1 {
+    font-family: Roboto, sans-serif;
```

Comment:
The font family is already defined in the `body` in the `index.scss` file. So, there is no need to define it again here. Please remove this line.